app/src/main/java/fr/gaulupeau/apps/Poche/data/WallabagDbOpenHelper.java

```diff
@@ -51,6 +53,21 @@ public void onUpgrade(Database db, int oldVersion, int newVersion) {
                     }
                 }
 
+                if (oldVersion < 103) {
+                    Log.i(TAG, "Migrating to version " + 103);
+
+                    ArticleContentDao.createTable(db, false);
+
+                    db.execSQL("insert into " + ArticleContentDao.TABLENAME +
+                            "(" + ArticleContentDao.Properties.Id.columnName +
+                            ", " + ArticleContentDao.Properties.Content.columnName + ")" +
+                            " select " + ArticleDao.Properties.Id.columnName + ", CONTENT" +
```

Comment:
That's a bit complicated. The basic idea is that old names (the hardcoded `CONTENT` here) will **never** change (because we can't retroactively change DB in older app versions). But the field `ArticleContentDao.Properties.Content.columnName` may actually change its value in a different version (column being renamed).

Actually, using ~constants~fields (that may change) in migrations is a bit dubious, because future field changes may affect old migrations. But it can be dealt with. Plus using fields lowers the probability of typos (except for not well readable string concatenations).